hash/ibm5170.xml

```diff
@@ -5483,8 +5485,8 @@
 		<info name="version" value="6.0" />
 		<info name="release" value="19931125"/>
 		<part name="flop1" interface="floppy_3_5">
-			<dataarea name="flop" size="1474560">
-				<rom name="mitsumi mouse driver 6.0.img" size="1474560" crc="591804ee" sha1="bb41f6447f631a4efe850431be3c720f9cadc4d1"/>
+			<dataarea name="flop" size="184320">
+				<rom name="mitsumi mouse driver 6.0.img" size="184320" crc="05bd4b0e" sha1="7a80b55a2d86a04aa19464b69cd0d524da7ef60c"/>
```

Comment:
This is a 3.5" disk and according to the header it's 720KB. So the .img file ought to be 737280 bytes, even though it's not full.

Reply:
Is that a must with the img standard? The disk looks like this. There aren't even sectors above 180.
![Mitsumi](https://ia802902.us.archive.org/24/items/MitsumiMouseDriverV6.03.5IBMPC/Mouse%20Driver%5BPC%5D.png)

Reply:
Oh, weird!

The problem is, MAME won't load it like this:

```
Fatal error: Device 3.5" double density floppy drive load failed: Unable to iden
tify the image format
```

Reply:
Yup, it checks the image size and won't load it if it's not a known size.  I think most other emulators are like that too.  If you don't want to pad it then make it an image that has explicit sector numbering like IMD.